crates/astria-conductor/src/block_verifier.rs

```diff
@@ -121,7 +121,7 @@ impl BlockVerifier {
         // in the block is for the previous height
         let validator_set = self
             .sequencer_client
-            .get_validator_set(height - 1)
+            .validators(height - 1, tendermint_rpc::Paging::Default)
```

Comment:
same here

Reply:
This is probably correct (`height -1` to `height - 1`), whereas the other call above indeed changes `height - 1` to `height`.